rest/route/agent.go

```diff
@@ -1183,6 +1183,59 @@ func (h *gitServePatchHandler) Run(ctx context.Context) gimlet.Responder {
 	return gimlet.NewJSONResponse(p)
 }
 
+// GET /task/{task_id}/git/version
+type gitServeVersionHandler struct {
+	taskID    string
+	versionID string
+}
+
+func makeGitServeVersion() gimlet.RouteHandler {
+	return &gitServeVersionHandler{}
+}
+
+func (h *gitServeVersionHandler) Factory() gimlet.RouteHandler {
+	return &gitServeVersionHandler{}
+}
+
+func (h *gitServeVersionHandler) Parse(ctx context.Context, r *http.Request) error {
+	if h.taskID = gimlet.GetVars(r)["task_id"]; h.taskID == "" {
+		return errors.New("missing task ID")
+	}
+	if versionParam, exists := r.URL.Query()["version"]; exists {
```

Comment:
Rather than requesting arbitrary versions, it seems like we should just use the task to look up its own version. I don't think we would want this route to serve arbitrary versions, just the task's own version.